### DISCUSSION TITLE
Detecting client side if the current domain is in domainLock

### DISCUSSION THREAD

**Discussion by Morusec** [javascript-obfuscator/javascript-obfuscator#837] [Q&A] [answered]
Hi!

I was wondering if there was a way to detect, client side, if the current domain is listed in the whitelisted domainLock domains. 
Just trying to avoid having to maintain a list of domains in javascript, I though maybe there was an internal Javascript variable I could poke to test the domain's validity.

The idea is to show a clear message about the error instead of just breaking.

Thank you for this great product!

**Comment by da411d** [comment, marked as answer]
Take a look upgraded [domainLock](https://github.com/javascript-obfuscator/javascript-obfuscator#domainlock) and [domainLockRedirectUrl](https://github.com/javascript-obfuscator/javascript-obfuscator#domainlockredirecturl) options.

So you can redirect users to some url with the message instead of just breaking=)
